src/main/java/use_case/TaskGetter.java

```diff
@@ -42,4 +42,14 @@
 
         return task_data;
     }
+
+    public Task getTaskByName(String name, AccessTodoData accessTodoData) {
```

Comment:
Are AccessTodoData and TaskGetter in the same layer (or say, they will be used by same layer)?

Reply:
yeah they're both in the use case layer